explainability-service/src/main/java/org/kie/trustyai/service/validators/metrics/fairness/group/GroupMetricRequestValidator.java

```diff
@@ -30,53 +30,79 @@ public void initialize(ValidGroupMetricRequest constraintAnnotation) {
     @Override
     public boolean isValid(GroupMetricRequest request, ConstraintValidatorContext context) {
```

Comment:
This method is becoming quite big, I would consider to refactor it to split output and attribute validation in a submethod.
Wdyt?
Btw it is fine to do in a follow-up ticket

Reply:
Done